libmachine/host.go

```diff
@@ -379,9 +381,17 @@ func getHostState(host Host, hostListItemsChan chan<- HostListItem) {
 
 	dockerHost := os.Getenv("DOCKER_HOST")
 
+	normalActive := dockerHost == url && currentState != state.Stopped
+
+	replacedSwarmPort := strings.Replace(url, ":2376", ":3376", -1)
+
+	// TODO: HUGE HORRIBLE HACK!!!! DO NOT HARDCODE DOCKER PORT 2376 OR SWARM PORT 3376
```

Comment:
While this _is_ horrible, we're hardcoding ports in a bajillion other places already, and given that there isn't a way to set a custom port yet, it's probably fine to just leave this :)